central/billingmetrics/store/postgres/store.go

```diff
@@ -0,0 +1,150 @@
+package postgres
+
+// Code is not generated by pg-bindings generator. DO EDIT.
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/metrics"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+	ops "github.com/stackrox/rox/pkg/metrics"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	pkgSchema "github.com/stackrox/rox/pkg/postgres/schema"
+	"github.com/stackrox/rox/pkg/protoconv"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+const (
+	baseTable = pkgSchema.BillingMetricsTableName
+
+	getStmt    = "SELECT ts, serialized FROM " + baseTable + " WHERE ts >= $1 AND ts < $2"
+	insertStmt = "INSERT INTO " + baseTable + "(ts, serialized) VALUES ($1, $2)"
+)
+
+var (
+	log            = logging.LoggerForModule()
+	schema         = pkgSchema.BillingMetricsSchema
+	targetResource = resources.Administration
+	zeroTime       = time.Unix(0, 0).UTC()
```

Comment:
zeroTime is also `zeroTime time.Time`

Reply:
No, it's not. `time.Time` is "local" zero.

Reply:
You're right, I'm wrong.